### ISSUE TITLE
Calcite-switch has no selected state color in high contrast mode

### ISSUE THREAD

**Issue by mpriour** [Esri/calcite-components#3987]
### Summary

The switched or checked state for `calcite-switch` is not visible in any browser when a high contrast accessibility theme is selected on a Windows computer. The lack of color makes it hard to determine what state the switch is in, unless there are obvious labels on either side of the switch.

This issue was discovered as part of #3887 - [High contrast audit](https://github.com/Esri/calcite-components/wiki/High-Contrast-Audit#switch) and identified as a severity level 3 out of 4 issue.

### Actual Behavior

There is no selection color in high contrast mode.

<img src="https://user-images.githubusercontent.com/142636/149253593-005858c8-e9ba-40b4-b733-3518c6a5bf2a.png" />

### Expected Behavior

The selection should be obvious like it is when the OS is in "normal" mode

<img src="https://user-images.githubusercontent.com/142636/149253558-b1889b58-56ba-4a0c-a741-46c5d9e274bf.png" />

### Reproduction Sample

https://esri.github.io/calcite-components/?path=/story/components-controls-switch--wrapping-calcite-label

### Reproduction Steps

1. Set the theme on a Windows machine or vm instance to high contrast accessibility.
2. View the above sample. Nothing is shown in the switch track.

### Reproduction Version

@esri/calcite-components@1.0.0-75

### Relevant Info

OS: Windows 7, 10, or 11 with a high contrast theme selected
Browser: Any (Chrome, Edge, Firefox)

### Source

- [X] CDN
- [X] NPM package

**Comment by mpriour** [comment]
There are a few options for the checked state of the switch track in high contrast mode:

* `linkText` system color - `linkText` should always be a distinctly different color that normal text and have good contrast with the window's background color.
<img width="550" alt="switch-linktext" src="https://user-images.githubusercontent.com/142636/150838291-9f738523-5472-4c94-a4c5-5a27a2c0bdd1.png">

* `canvasText` system color - `canvasText` will match the normal text and the outline of the switch and the switch handle.
<img width="550" alt="switch-canvastext" src="https://user-images.githubusercontent.com/142636/150838935-54b34df3-ce56-4991-a6c5-d5552caeebbb.png">

* `highlightText` system color - `highlightText` should be a nearly opposite color from the window background.
<img width="550" alt="switch-highlighttext" src="https://user-images.githubusercontent.com/142636/150839392-604d3e5c-f5fd-4d7b-86d2-9a0350d9ea38.png">

Any opinions welcomed.

**Comment by benelan** [comment]
cc @macandcheese

**Comment by macandcheese** [comment]
IMO the `canvasText system color` is the best option here.

**Comment by driskull** [comment]
It seems like we need to setup screener tests for high contrast mode. @eriklharper @benelan @jcfranco

**Comment by eriklharper** [comment]
Perfect use case!

**Comment by mpriour** [comment]
Thanks @macandcheese, that matches the color I used in the radio button high contrast fix.

**Comment by MehulChoksey** [comment]
+1 for `canvasText system color `

**Comment by benelan** [comment]
verified on beta.77
